graylog2-web-interface/src/views/components/searchbar/bookmark/BookmarkList.test.jsx

```diff
@@ -0,0 +1,150 @@
+            .title('test')
+            .description('desc')
+            .build(),
+        ],
+      };
+      const wrapper = mount(<BookmarkList toggleModal={() => {}}
+                                          showModal
+                                          deleteBookmark={() => {}}
+                                          views={views} />);
+      expect(wrapper.find('ModalDialog')).toMatchSnapshot();
+    });
+
+    it('should handle toggleModal', () => {
+      const views = {
+        pagination: {
+          total: 0,
+          page: 0,
+          perPage: 5,
+        },
+        list: [],
+      };
+      const wrapper = mount(<BookmarkList toggleModal={() => {}}
+                                          showModal
+                                          deleteBookmark={() => {}}
+                                          views={views} />);
+      expect(wrapper.find('ModalDialog')).toMatchSnapshot();
+    });
+
+    it('should handle with toggle modal', () => {
+      const onToggleModal = jest.fn();
+      const views = {
+        pagination: {
+          total: 0,
+          page: 0,
+          perPage: 5,
+        },
+        list: [
+          View.builder()
+            .id('foo-bar')
+            .title('test')
+            .description('desc')
+            .build(),
+        ],
+      };
+
+      const wrapper = mount(<BookmarkList toggleModal={onToggleModal}
+                                          showModal
+                                          deleteBookmark={() => {}}
+                                          views={views} />);
+      wrapper.find('button[children="Cancel"]').simulate('click');
+      expect(onToggleModal).toBeCalledTimes(1);
+    });
+
+    it('should handle with delete', () => {
+      window.confirm = jest.fn(() => true);
+      const onDelete = jest.fn(() => {
+        return new Promise(() => {});
+      });
+      const views = {
+        pagination: {
+          total: 0,
+          page: 0,
+          perPage: 5,
+        },
+        list: [
+          View.builder()
+            .id('foo-bar')
+            .title('test')
+            .description('desc')
+            .build(),
+        ],
+      };
+
+      const wrapper = mount(<BookmarkList toggleModal={() => {}}
+                                          showModal
+                                          deleteBookmark={onDelete}
+                                          views={views} />);
+      wrapper.find('button.list-group-item').simulate('click');
+      wrapper.find('button[children="Delete"]').simulate('click');
+      expect(window.confirm).toBeCalledTimes(1);
+      expect(onDelete).toBeCalledTimes(1);
+    });
+
+    it('should handle with load', () => {
+      const onLoad = jest.fn(() => { return new Promise(() => {}); });
+      const views = {
+        pagination: {
+          total: 0,
+          page: 0,
+          perPage: 5,
+        },
+        list: [
+          View.builder()
```

Comment:
I think the `View` fixture creation should be extracted into a constant or function to prevent repeating yourself (and to make it easier to apply changes like adding another mandatory property in the future).